### PR TITLE
[WIP] Populate Repo with basic functions and examples

### PR DESCRIPTION
**Pull request by pmyers16**
This PR is meant to set up the repository for use for multiple people. It will populate with initial functions and example scripts.

Ports over necessary functions from ezscalp and eztrack so that there are no private repo dependencies.

### CODE REVIEW

**Review by adam2392 — APPROVED** [review]
Do you have the boilerplate BIDS conversion scripts for:

1. source -> raw?
2. EEGLAB ICA -> BIDS deriv? (or is this done in MATLAB?)

**Comment by pmyers16** [comment]
> Do you have the boilerplate BIDS conversion scripts for:
> 
> 1. source -> raw?
> 2. EEGLAB ICA -> BIDS deriv? (or is this done in MATLAB?)

Put up source -> raw. Will work on ICA -> bids, but I think they can work with the non-ica data for now

File: episcalp/bids/utils.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    )  # remove dashes from channel names
+    raw = raw.rename_channels(
+        lambda x: x.replace("`", "'")
+    )  # remove dashes from channel names
+    raw = raw.rename_channels(lambda x: _reformatchanlabel(x))
+
+    return raw
+
+
+def preprocess_raw(raw, bids_path, montage):
```
Comment (adam2392) [inline episcalp/bids/utils.py:64]: Can you refactor this to be named:

`bids_preprocess_raw`, or `channel_label_process_raw`? 

`preprocess_raw` sounds like a wrapper for preprocessing the raw data with filtering/referencing/bad channel dropping/etc.

**Review by adam2392 — APPROVED** [review]
one comment, and otw LGTM.

**Pull request merged**